be/src/fs/fs_starlet.cpp

```diff
@@ -250,8 +250,9 @@ class StarletFileSystem : public FileSystem {
         if (!fs_st.ok()) {
             return to_status(fs_st.status());
         }
-
-        auto file_st = (*fs_st)->open(pair.first, ReadOptions{.skip_fill_local_cache = opts.skip_fill_local_cache});
+        auto opt = ReadOptions();
+        opt.skip_fill_local_cache = opts.skip_fill_local_cache;
```

Comment:
If we remove the declaration `RandomAccessFileOptions() = default`, then we needn't modify this.

Reply:
`ReadOptions` is in `starlet`

Reply:
Why in this way? Does not C++20 support it?

Reply:
as ReadOptions is in starlet, if we want keep the previous code, we should change that in starlet (remove ReadOptions() = default; )